AUTHORS

```diff
@@ -0,0 +1,35 @@
+# List of AUTHORS who contributed over time to the AzerothCore project
+
+## Warning
+The code of AzerothCore is shipped as it is without any form of warranty,
+and - except for third party libraries licensed
+- under the GPL 2 for old files which you can read from the file "LICENSE-GPL2"
+- under the AGPL 3 for new files which you can read from the file "LICENSE-AGPL3"
+
+## Point of current development
+The project is currently hosted at https://www.azerothcore.org/ and developed under https://github.com/azerothcore
+
+## History of development
+Development of this project dates back to 2004, and was developed under various umbrellas over time:
+* WoW Daemon Team, 2004
+* MaNGOS project, 2005-2008, located at http://www.mangosproject.org
+* MaNGOS project, 2008-2011, located at http://getmangos.com
+* SD2 project, 2008-2009, located at http://getmangos.com
+* TrinityCore, 2008-2021, located at https://www.trinitycore.org/
+* SunwellCore, located at https://www.azerothcore.org/pages/sunwell.pl/
+* AzerothCore, 2016-2021, located at https://www.azerothcore.org/
+
+## Authorship of the code
+Authorship is assigned for each commit within the git history, which is stored in these git repositories:
+* github.com/cmangos/mangos-svn (History from MaNGOS project from 2005-2008, originally hosted at http://mangos.svn.sourceforge.net)
+* github.com/TrinityCore/TrinityCore
+* github.com/azerothcore/azerothcore-wotlk
+
+Unfortunately we have no detailed information of the history of the WoWD project;
+if somebody can provide information, please contact us, so that we can make this history available
```

Comment:
we should also underline how there is no git history of sunwellcore

Reply:
Added, need check

Reply:
Maybe we can just add Xinef and Pussywizzard for swc. That way we get some info about it